processing/src/main/java/io/druid/segment/DimensionSelectorUtils.java

```diff
@@ -246,4 +248,22 @@ public static BitSet makePredicateMatchingSet(DimensionSelector selector, Predic
     }
     return valueIds;
   }
+
+  public static DimensionSelector constantSelector(final String value)
```

Comment:
Please annotate the param as `@Nullable`

Reply:
Changed.